runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpTrackerPlugin.java

```diff
@@ -62,6 +57,13 @@
 import net.runelite.client.ui.overlay.OverlayManager;
 import net.runelite.client.util.ImageUtil;
 import net.runelite.http.api.xp.XpClient;
+import net.runelite.api.Client;
```

Comment:
Imports should be sorted, update your formatter with the xml from the wiki

https://github.com/runelite/runelite/wiki/Code-Conventions